src/main/java/trackitnus/logic/commands/module/EditModuleCommand.java

```diff
@@ -103,15 +100,21 @@ public boolean equals(Object other) {
             && editModuleDescriptor.equals(e.editModuleDescriptor);
     }
 
+    @Override
+    public String toString() {
+        return "EditModuleCommand{" +
+            "code=" + code +
+            ", editModuleDescriptor=" + editModuleDescriptor +
+            '}';
+    }
+
```

Comment:
Why do we need this toString method? Maybe change to `Code: ` and `editModuleDescriptor: ` instead of `code=` and `editModuleDescriptor=`

Reply:
this the auto toString method generated by IntelliJ. It will be used for testing only so I guess it's best to stick to the default format

Reply:
Oh I didnt IntelliJ will autogenerate toString methods. Do you plan on removing this? Or keep it just for testing

Reply:
No it's a good thing to keep, why need to remove right?

Reply:
Ok